tests/php/Controller/RoomControllerTest.php

```diff
@@ -1,167 +0,0 @@
-<?php
```

Comment:
no more unit test or is this covered by the command line integration tests already ?

Reply:
No, by the millions of api integration tests which are untouched